docs/src/content/docs/hi/showcase.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Starlight प्रदर्शन
+description: Starlight और सामुदायिक टूल से निर्मित साइटों की खोज करें जो Starlight का विस्तार करते हैं!
+---
+
+:::tip[अपनी स्वयं की साइट जोड़ें!]
+क्या आपने Starlight साइट या Starlight के लिए कोई उपकरण बनाया है?
+इस पेज पर एक लिंक जोड़कर एक PR खोलें!
+:::
+
+## Sites
```

Comment:
Just to double check, this was not translated on purpose and not something that was missed?

Reply:
ohh wow I completely missed it :man_facepalming: I can fix it but a suggestion will be quicker since I am outside (`साइटें` is satisfactory here)